### DISCUSSION TITLE
نمایش پیشفرض

### DISCUSSION THREAD

**Discussion by hassanzadeh88** [majidh1/JalaliDatePicker#53] [Question] [answered]
چطوری میشه وقتی ک صفحه لود میشه تقویم ب صورت پیشفرض باز باشه و بعد از انتخاب هم بسته نشه و همچنان باز بمونه؟

**Comment by majidh1** [comment, marked as answer]
از نمونه زیر میتونید استفاده کنید. فقط باید از ورژن 0.9.3 استفاده کنید

https://jsfiddle.net/ba53gtzp/3/
تنظیمات زیر کمکتون میکنه
	autoHide: false,
	hideAfterChange:false,
	showCloseBtn:false

**Reply by hassanzadeh88** [reply]
خیلی ممنون از راهنماییتون ی سوال دیگه هم ک دارم این هست ک این تقویم قابلیت انتخاب چندتایی نداره؟یا وقتی روی تقویم کلیک میکنیم اعدادش رنگی بمونن؟

**Comment by majidh1** [comment]
خواهش میکنم ممنون از مشارکتتون 
نه قابلیت رنج تاریخ نداره این دیتپیکر
روز انتخابی کاربر بک آبی میگیره

**Comment by hassanzadeh88** [comment]
من کلا انتخاب چندتایی رو با ویو جی اس برای خودم حلش کردم سوالی ک دارم اینه ک من چطوری میتونم ب مقدار jdp-day استایل دهی کنم؟چون هرطوری میخام بگم ک روی هر عددی کلیک شد یه استایل مشخصی رو بهش بده 
من از نسخه قبلتر استفاده میکنم و روز انتخابیم هیج رنگی نمیگیره

**Comment by majidh1** [comment]
با این سلکتور میتونید روز انتخاب شده رو با css انتخاب کنید. بهتره از ورژن آخر استفاده کنید
```css
jdp-container .jdp-day.selected {
    background-color: red!important;
    color: #fff!important;
}
```
